packages/shared-http-pieces/src/lib/influx.ts

```diff
@@ -0,0 +1,22 @@
+import { container } from '@skyra/http-framework';
+import { Client } from '@skyra/influx-utilities';
+
+export function isInfluxInitialized() {
+	return [container.analytics, container.analytics?.writeApi, container.analytics?.queryApi].every(Boolean);
+}
```

Comment:
Only one thing needs to be checked, and I'd explore the possibility to cast `container.analytics` to non-undefined.